### PR TITLE
{Packaging} Update version management policy

### PR DESCRIPTION
**Pull request by fengzhou-msft**
Updated documentation for our version management policy.

Our next release version will be 2.1.0. Extended the expiration version to 3.0.0. Feel free to remove the deprecated parameters if you think they are no longer needed.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: doc/version_management.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Current Norm
+Starting from version 2.1.0, Azure CLI updates:  
+* MAJOR version for core changes that break commands behavior globally;  
+* MINOR version for general backward compatible feature changes and service commands breaking changes;  
+* PATCH verison for bug fixes.
+
+## Backward Compatibility
+Considering Azure CLI is a command line tool for Azure Services, we tend to just bump the MINOR verison for breaking changes in a service module command. All breaking changes for a command will be marked as **BREAKING CHANGE** in [release notes](https://docs.microsoft.com/cli/azure/release-notes-azure-cli?view=azure-cli-latest).
+
+From command level, packages only upgrading PATCH version guanrantees backward compatibilty.
```
Comment (jiasli) [inline doc/version_management.md]:
```suggestion
At command level, packages only upgrading the PATCH version guarantee backward compatibility.
```

File: doc/version_management.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Azure CLI 2 was used for branding as compared to Azure classic CLI. We intended to keep the MAJOR version fixed at 2 for a while. A regular release only updates the PATCH version, which was a little confusing.
+
+## Current Norm
+Starting from version 2.1.0, Azure CLI updates:  
+* MAJOR version for core changes that break commands behavior globally;  
+* MINOR version for general backward compatible feature changes and service commands breaking changes;  
+* PATCH verison for bug fixes.
+
+## Backward Compatibility
+Considering Azure CLI is a command line tool for Azure Services, we tend to just bump the MINOR verison for breaking changes in a service module command. All breaking changes for a command will be marked as **BREAKING CHANGE** in [release notes](https://docs.microsoft.com/cli/azure/release-notes-azure-cli?view=azure-cli-latest).
```
Comment (jiasli) [inline doc/version_management.md]:
```suggestion
Considering Azure CLI is a command line tool for Azure Services, we tend to just bump the MINOR version for breaking changes in a command module. All breaking changes for commands will be marked as **BREAKING CHANGE** in [release notes](https://docs.microsoft.com/cli/azure/release-notes-azure-cli?view=azure-cli-latest).
```

File: doc/version_management.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Azure CLI Version Management
+============================
+
+## History
+Azure CLI 2 was used for branding as compared to Azure classic CLI. We intended to keep the MAJOR version fixed at 2 for a while. A regular release only updates the PATCH version, which was a little confusing.
+
+## Current Norm
+Starting from version 2.1.0, Azure CLI updates:  
+* MAJOR version for core changes that break commands behavior globally;
```
Comment (jiasli) [inline doc/version_management.md]:
```suggestion
* MAJOR version for core changes that break commands' behavior globally;  
```

File: doc/version_management.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+Azure CLI Version Management
+============================
+
+## History
+Azure CLI 2 was used for branding as compared to Azure classic CLI. We intended to keep the MAJOR version fixed at 2 for a while. A regular release only updates the PATCH version, which was a little confusing.
+
+## Current Norm
+Starting from version 2.1.0, Azure CLI updates:  
+* MAJOR version for core changes that break commands behavior globally;  
+* MINOR version for general backward compatible feature changes and service commands breaking changes;
```
Comment (jiasli) [inline doc/version_management.md]:
```suggestion
* MINOR version for general backward compatible feature changes and service command breaking changes;  
```

**Comment by fengzhou-msft** [comment]
Code owners, please help review this PR. need your approval to merge PR. @kalyanaj @haroldrandom @MyronFanQiu @qwordy

File: doc/version_management.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+Azure CLI Version Management
+============================
+
+## History
+Azure CLI 2 was used for branding as compared to Azure classic CLI. We intended to keep the MAJOR version fixed at 2 for a while. A regular release only updates the PATCH version, which was a little confusing.
```
Comment (qwordy) [inline doc/version_management.md]: "A regular release only updates the PATCH version, which was a little confusing."
Maybe we should use same tense.

Comment (fengzhou-msft) [reply doc/version_management.md]: updated

**Pull request merged**